types/shared.d.ts

```diff
@@ -12,12 +12,7 @@ export interface PageData {
   lastUpdated?: number
 }
 
-export interface Header {
-  level: number
-  title: string
```

Comment:
@brd-cc Check here, the original property is `title`. It is transformed to `text` in the `mapHeaders` method. :eyes: